components/layout/TopBar.tsx

```diff
@@ -184,7 +184,11 @@ export default function TopBar({
 
   return (
     <Container>
-      <StyledLogo src={dwollaDevLogo} alt="" />
+      <Link href="/">
+        <a>
+          <StyledLogo src={dwollaDevLogo} alt="" />
+        </a>
+      </Link>
```

Comment:
Realized the link was added only to the side nav, so added it to the (mobile) top bar and footer as well